docs/source/make_a_release.rst

```diff
@@ -179,3 +179,44 @@ Step-by-step
       rm -rf dist/* && \
       rmvirtualenv pynwb-${release}-release && \
       rmvirtualenv pynwb-${release}-install-test
+
+-------------
+Conda Release
+-------------
+
+In order to release a new version on conda, follow the steps below:
+
+1. Clone feedstock
+
+   .. code::
+
+      $ git clone git@github.com:conda-forge/pynwb-feedstock.git
+
+
+2. Create a new branch
+
+   .. code::
+
+      $ cd pynwb-feedstok
+      $ git checkout -b new_version
+
+
+3. Modify meta.yaml
```

Comment:
Since these step should make it easy to release, can we have an automated a way to compute the sha256 ?

Since the conda release will be done following up the PyPI release, the source distribution should be available in the same folder ...